packages/rancher-backup/charts/app-README.md

```diff
@@ -0,0 +1,34 @@
+# Rancher Backup
+
+This chart provides ability to back up and restore the Rancher application running on any Kubernetes cluster.
+Refer [this](https://github.com/rancher/backup-restore-operator) repository for implementation details.
+
+### Configuration
```

Comment:
Should the `Configuration` and `CRDs` section should be omitted here since they are already discussed in the `README.md`?

Reply:
Since the purpose of the `app-README.md` is to be the first page that a user sees when installing a chart via `Apps & Marketplace`, I think it should be more high-level about what the chart is (i.e. how you would summarize `rancher-backup` to someone in person) rather than how to configure it, which is already covered in the `README.md`.

Reply:
For example, for `rancher-backup` it might make sense to add something similar to the [User Flow](https://github.com/rancher/backup-restore-operator#user-flow) section in the repo's README.md. 

Something that explains how the chart actually provides the ability to backup and restore Rancher by 1. Using the default `ResourceSet` to identify which resources need to be backed up 2. creating a `Backup` (also should explain what storage locations are supported e.g. S3, PV, etc.) and 3. creating a `Restore` to actually perform the backup.

Reply:
yeah I was thinking of something similar, that's why I thought I'll mark the PR as draft and edit it more

Reply:
yeah. we could remove this so it looks nicer and is more high level descrition of the chart since the Helm README is available inside the app deployment.

Reply:
@deniseschannon @aiyengar2 the PR is still Draft/WIP, I'll request review once it's ready